src/ExposureHistory/detail/ExposureActions.tsx

```diff
@@ -41,7 +42,13 @@ const ExposureActions: FunctionComponent = () => {
               healthAuthorityName,
             })}
           </GlobalText>
-          <View style={style.recommendations}>
+          <View
+            style={
+              screenWidth > 350
+                ? [style.row, style.recommendations]
```

Comment:
Thoughts on making `row` the default? Something like:

```
? { ...style.recommendations }
: { ...style.recommendations, ...style.recommendationsColumn }
…
recommendations: {
  display: "flex",
  flexDirection: "row",
  …
recommendationsColumn: {
  flexDirection: "column"
}
```

I also prefer the object spread style but totally up to personal preference!